components/pedigree/ui/src/main/resources/PhenoTips/UIX_Field__pedigree.xml

```diff
@@ -126,32 +136,17 @@
       pedigreeTrigger.observe("click", function(event) {
         event.stop();
         document.observe("xwiki:document:saved", function () {
-          window.self.location = pedigreeTrigger.href;
```

Comment:
I can't remember why I asked. On a different note, why did you chose to hardcode ```location.search``` to ```?sheet=...```